src/reducers/application.js

```diff
@@ -3,6 +3,7 @@ import { NETWORK_TYPES } from "../config";
 
 export const initialState = {
   isLoading: false,
+  isVerifying: false,
```

Comment:
in term of implementation
- isVerifying is a very name ambiguous, as much as isLoading (what is loading ? what is being verified ?)
- this being part of the main reducer (it looks like it is) is slightly weird => supposed to be part of revoke feature right ?

Suggestion (pick the one you want)
- move to a more specific reducer (just code organization nothing much to change)
- use hooks and lift down state in components
- use hooks and keep it in redux (not that can stick with redux, redux has a nice useSelector and useDispatch hook)

Note: I'm talking about hooks because I think you started an MR using it somewhere

Reply:
Updated. Please have a look and see if can be improved.